sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -269,6 +270,9 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
       logInfo(s"The number of concurrent GPU tasks allowed is $concurrentGpuTasks")
       GpuSemaphore.initialize(concurrentGpuTasks)
     } catch {
+      case e: CudaFatalException =>
+        logError("Exception in the executor plugin, shutting down!", e)
+        logGpuDebugInfoAndExit(1)
```

Comment:
this exit code is 1, whilst the driver-side one is 20. Mostly curious on why the difference.

Reply:
I didn't want to change the exit codes that were there previously.